### PR TITLE
Add example for Unicode Code point literal > 255

### PR DESCRIPTION
**Pull request by juergenhoetzel**
Only Basic Latin and Latin-1 Supplement code points fit into a single byte.

### CODE REVIEW

**Comment by chrboesch** [comment]
You also need to rebuild the patch file because you have added new lines.

File: exercises/059_integers.zig [hunk 1]
```diff
@@ -6,7 +6,8 @@
 //     const a2: u8 = 0x41;      // hexadecimal
 //     const a3: u8 = 0o101;     // octal
 //     const a4: u8 = 0b1000001; // binary
-//     const a5: u8 = 'A';       // UTF-8 code point literal
+//     const a5: u8 = 'A';       // ASCII code point literal
+//     const a6: u16 = '⚡'      // Unicode code points can take up to 21 bits
```
Comment (chrboesch) [inline exercises/059_integers.zig:10]: There is a missig space before the comment.

Comment (juergenhoetzel) [reply exercises/059_integers.zig]: Thanks: my monospace font took more space for this glyph :thinking: . I have chosen another character in the amended commit and also added the missing ";".

**Comment by juergenhoetzel** [comment]
> You also need to rebuild the patch file because you have added new lines.

Fixed in the amended commit. 

BTW. Some other patches are also "out-of-sync": 
```

Healing 047_methods...
patching file patches/healed/047_methods.zig (read from exercises/047_methods.zig)
Hunk #1 succeeded at 91 (offset -1 lines).
---
Healing 060_floats...
patching file patches/healed/060_floats.zig (read from exercises/060_floats.zig)
Hunk #1 succeeded at 46 (offset 3 lines).

```

**Comment by chrboesch** [comment]
> BTW. Some other patches are also "out-of-sync":

Fixed, thank you! :smile:

**Pull request merged**